LOW 2 - plakaty na mieście, Fatal Five-Way (#1395)
tpwres/teczki-polskiego-wrestlingu: content/e/low/2025-04-06-low-2.md

* On 31.03.2025 Legacy confirmed [Sinister](@/w/sinister.md) vs [Syriusz Dziedzic](@/w/dziedzic.md), and announced that it would be another Road to the Title Match.
* Later that same day, Legacy released a [video][marceli-dinozaur] in which [Marcelito](@/w/marcelito.md) states his new goal is to "turn off the lights" of every legend of Polish wrestling, and calls for a fight against "a real legend, a veteran, a dinosaur of the Polish wrestling scene" - implying his match could be against David Oliwa.
* The next day Legacy officially confirmed [David Oliwa](@/w/david-oliwa.md) vs [Marcelito](@/w/marcelito.md).

> * On 4.04.2025 Legacy announced an eleventh hour addition to the card - a Fatal Five-Way match between [Shadow](@/w/shadow.md), [Biesiad Strong](@/w/biesiad.md), [Aron Wake](@/w/aron-wake.md), [Sambor](@/w/sambor.md) and an unknown fifth opponent. Originally planned as a 4-way, it was expanded after a "familiar face", who wished to remain anonymous for the time being, contacted the promotion and suggested adding the fifth person to the match. Due to Biesiad's arrest by PM Shooter henchmen at [PpW Teraz Albo Nigdy](@/e/ppw/2025-03-15-ppw-teraz-albo-nigdy.md), the match might return to the 4-way formula if he fails to appear.

Reviewer: ```suggestion
* On 4.04.2025 Legacy announced an eleventh hour addition to the card - a Fatal Five-Way match between [Shadow](@/w/shadow.md), [Biesiad Strong](@/w/biesiad.md), [Aron Wake](@/w/aron-wake.md), [Sambor](@/w/sambor.md) and an unspecified fifth opponent. Originally planned as a 4-way, it was expanded after a "familiar face", who wished to remain anonymous for the time being, contacted the promotion and suggested adding the fifth person to the match. Due to Biesiad's arrest by PM Shooter henchmen at [PpW Teraz Albo Nigdy](@/e/ppw/2025-03-15-ppw-teraz-albo-nigdy.md), the match might return to the 4-way formula, should he fail to appear.
```
